user_tools/src/spark_rapids_pytools/cloud_api/dataproc.py

```diff
@@ -337,7 +337,11 @@ def parse_accelerator_description(raw_description: str) -> dict:
                 parsing_res.setdefault('gpu_mem', gpu_device.get_gpu_mem()[0])
             return parsing_res
 
-        accelerator_arr = self.props.get_value_silent('accelerators')
+        try:
+            accelerator_arr = self.props.get_value_silent('accelerators')
```

Comment:
so I'm 99% sure I put this here because I saw an issue with it. Normally I wouldn't have just added it.

I don't remember specifics but get_value_silent first calls:

`        return get_elem_non_safe(self.props, key_strs)
`
  so maybe self.props was empty..  not sure but at this point would rather leave for now.  We can mark this as followup if we think its important

Reply:
Thanks @tgravescs. We can keep it as it is for now.